client/src/models/model.ts

```diff
@@ -0,0 +1,14 @@
+export enum Role {
+    SuperAdmin = "SuperAdmin",
+    Admin = "Admin",
+    Subscriber = "Subscriber"
+  }
+
+export interface UserAction{
+  APIData:Array<object>,
+  UsersData:Array<Array<string>>,
+  addUser(refer:any):void,
+  selectedRowEdit(refer:any):void,
+  selectedRowDelete(i:any):void
```

Comment:
why using any type?
we should define proper types as far as possible.

Reply:
changed the any type with proper type